### PR TITLE
refactor:  replace laminas-mvc-console with laminas-cli

### PR DESCRIPTION
**Pull request by fibble**
## Description

Migrate laminas-mvc-console commands to laminas-cli

Related issue: [VOL-3615](https://dvsa.atlassian.net/browse/VOL-3615)

Requires simultaneous merge of olcs-logging && olcs-backend or will break builds.

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [x] Have you performed a self-review of the code
- [x] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: composer.json
Comment (JoshuaLicense) [inline composer.json:1]: Where are the `composer.lock` changes?

**Review by JoshuaLicense — CHANGES_REQUESTED** [review]
Can we avoid using environment variables please? That doesn't look ideal.

Instead, I recommend we either use `$_SERVER['argv']` or `PHP_SAPI` ([docs](https://www.php.net/manual/en/function.php-sapi-name.php)) to determine what type of request it is _and/or_ pass a `--container` ([docs](https://docs.laminas.dev/laminas-cli/intro/#usage)) to the CLI command and overwrite the `Request` factory to return a CLI specific request object.

File: composer.json
Comment (fibble) [reply composer.json:1]: ah, ill update and push the lock!

File: composer.lock
Comment (JoshuaLicense) [inline composer.lock:1]: Thanks for adding! Can we just update the dependencies that are required as part of this PR please.

`composer remove laminas/laminas-mvc-console` & `composer require laminas/laminas-cli` 🙏 

Also, during testing, we can point to the dev branch in `olcs-logging` - `composer require olcs/olcs-logging:dev-[branch]`

**Review by JoshuaLicense — COMMENTED** [review]
Looks good, will hold off approval until after testing though.

**Pull request merged**